index.js

```diff
@@ -605,10 +606,44 @@ export class Placement {
           visibility_change_listener
         );
 
+        let ad_rotation_listener = () => {
+          if (
+            placement.inViewport(placement.target) &&
```

Comment:
Do we actually want to only rotate it if it's in the view port? I feel like having it rotate off screen is actually a better UX, and it will only get viewed if it gets seen again.

Reply:
It's probably OK to rotate even if it is out of the viewport. Testing this is going to be a challenge...

Reply:
Heh yea, we can keep it in for now. Better to be more conservative. 👍